Origin: https://github.com/p12tic/cppreference-doc/commit/573e92315a4bc14230ba5bb930bad36035a67671
Description: This changes the file extension of the qhelpgenerator input file to use
 a .qhp extension instead of .xml. This is required because an update to
 qhelpgenerator now rejects the .xml extension.

--- a/Makefile
+++ b/Makefile
@@ -163,13 +163,13 @@
 #build the .qch (QT help) file
 output/cppreference-doc-en-cpp.qch: output/qch-help-project-cpp.xml
 	#qhelpgenerator only works if the project file is in the same directory as the documentation
-	cp "output/qch-help-project-cpp.xml" "output/reference/qch.xml"
+	cp "output/qch-help-project-cpp.xml" "output/reference/qch.qhp"
 
 	pushd "output/reference" > /dev/null; \
-	$(qhelpgenerator) "qch.xml" -o "../cppreference-doc-en-cpp.qch"; \
+	$(qhelpgenerator) "qch.qhp" -o "../cppreference-doc-en-cpp.qch"; \
 	popd > /dev/null
 
-	rm -f "output/reference/qch.xml"
+	rm -f "output/reference/qch.qhp"
 
 output/qch-help-project-cpp.xml: output/cppreference-doc-en-cpp.devhelp2
 	#build the file list
